### ISSUE TITLE
Blazor WebAssembly App template is not compiling

### ISSUE THREAD

**Issue by datvm** [aspnet/AspNetCore#14643]
### Describe the bug

I want to describe two problems: duplicate template items and created template is not compiling.

### To Reproduce
1. Create new Blazor project, there are two items with the same name and description (the 2nd one creates 3 projects, while the 3rd one is for client-only).

![image](https://user-images.githubusercontent.com/6388546/66001698-3e3a9a00-e4cc-11e9-8e48-547557198829.png)

2 . I want to use the 3rd one because I do not need API server, only front-end only app.

However, the created project cannot be compiled:

![image](https://user-images.githubusercontent.com/6388546/66001996-e6e8f980-e4cc-11e9-9807-4848d8d7880f.png)

I tried to update the package version to latest `preview9`, all other errors are gone except that NotFoundContent tag but new error are added:

![image](https://user-images.githubusercontent.com/6388546/66002162-42b38280-e4cd-11e9-8d65-a45cf9d749b6.png)

### Expected behavior
The template name and description should be updated, and the code updated to be able to compile.

### Screenshots
See above

### Additional context

I am using the latest Preview Visual Studio 16.4.0 Preview 1 on Windows 10 1903.

Same problem with 2nd template item, both package are in preview 6. I think somehow my VS has old template. Is there anyway to update it? I manually checked for update but everything is already up to date.

**Comment by datvm** [comment]
After some investigation, I was able to update the template, but the old ones still stay there. So somehow my VS setup wasn't able to update Blazor template.

I check [this article](https://docs.microsoft.com/en-us/aspnet/core/blazor/get-started?view=aspnetcore-3.0&tabs=visual-studio-code) and ran the following command to install the templates:

`dotnet new -i Microsoft.AspNetCore.Blazor.Templates::3.0.0-preview9.19465.2`

However, for my case, I had 401 error because I had multiple Nuget source (from Azure DevOps) and somehow Nuget decided to use the last one. Therefore I had to specify the source manually:

`dotnet new -i Microsoft.AspNetCore.Blazor.Templates::3.0.0-preview9.19465.2 --nuget-source "https://api.nuget.org/v3/index.json"`

Now in Visual Studio, the template list is populated with both old and new templates:

![image](https://user-images.githubusercontent.com/6388546/66003426-498fc480-e4d0-11e9-9cbe-d39c189fdf9f.png)

Would be appreciate to know how to clean up the old templates.
